### ISSUE TITLE
Support binary builds (like RVM does)

### ISSUE THREAD

**Issue by rkh** [postmodern/ruby-install#31]
That would be a killer feature.

**Comment by FranklinYu** [comment]
rvm/rvm#4298 could be related.

**Comment by postmodern** [comment]
Noticed that RVM's binaries haven't been updated recently. Not seeing binaries for newer versions of Ubuntu, Fedora, or OSX. https://rvm.io/binaries/ That seems like a blocker for average users not using ruby-install in an older VM or docker image.

**Comment by FranklinYu** [comment]
@postmodern Yes, and rvm/rvm#4298 has been talking about it. RVM team made barely any progress about this since last year though.

**Comment by rkh** [comment]
Note that rvm also downloads binaries from http://rubies.travis-ci.org/.

**Comment by FranklinYu** [comment]
@rkh Awesome! I’m curious whether they are path-dependent though. Will need some research.

**Comment by rkh** [comment]
I'm not working with Ruby in a professional setting at the moment, so I don't think I'll have time to look into this any time soon, sorry.
